.ci_support/lower_bound.yml

```diff
@@ -4,3 +4,5 @@ dependencies:
 - numpy =1.26.4
 - h5py =3.6.0
 - ase =3.23.0
+- semantikon =0.0.20
```

Comment:
The renaming of `typing` was only done in `semantikon==0.0.20` https://github.com/pyiron/semantikon/pull/170